seeders/20220509153850-admin.js

```diff
@@ -20,6 +20,125 @@ module.exports = {
+          isAssigned:"false",
+          passwordResetToken:"",
+          createdAt:new Date(),
+          updatedAt:new Date(),
+      },
+      {
+        
+        // id:100,
+        uuid:"9e3d2a6c-5484-4d86-8eb0-f0098cd7a540",
+        name:"Armstrong ngororano",
+        gender:"male",
+        idNumber:20040820,
+        district:"kicukiro",
+        sector:"kicukiro",
+        cell:"kicukiro",
+        email:"avengersphantom7l@gmail.com",
+        permitId:"qwerty12",
+        telNumber:"785278453",
+        carplate:"RAB322",
+        capacity:"10",
+        vehicletype:"ritco",
+        password:await bcrypt.hash("phantom123", 12),
+        roleId:7,
+        roleName:"operator",
+        isAssigned:"false",
+        passwordResetToken:"",
+        createdAt:new Date(),
+        updatedAt:new Date(),
+    },
+      {
+        
+          // id:101,
+          uuid:"9e3d2a6c-5484-4d86-8eb0-b0098cd7a540",
+          name:"Armstrong ngororano",
+          gender:"male",
+          idNumber:20040820,
+          district:"kicukiro",
+          sector:"kicukiro",
+          cell:"kicukiro",
+          email:"avengersphantom7b@gmail.com",
+          permitId:"qwerty12",
+          telNumber:"785278453",
+          carplate:"RAB322",
+          capacity:"10",
+          vehicletype:"ritco",
+          password:await bcrypt.hash("phantom123", 12),
+          roleId:1,
+          roleName:"driver",
+          isAssigned:"false",
+          passwordResetToken:"",
+          createdAt:new Date(),
+          updatedAt:new Date(),
+      },
+      {
+        
+          // id:102,
+          uuid:"9e3d2a6c-5484-4d86-8eb0-c0098cd7a540",
+          name:"Armstrong ngororano",
+          gender:"male",
+          idNumber:20040820,
+          district:"kicukiro",
+          sector:"kicukiro",
+          cell:"kicukiro",
+          email:"avengersphantom7c@gmail.com",
+          permitId:"qwerty12",
+          telNumber:"785278453",
+          carplate:"RAB322",
+          capacity:"10",
+          vehicletype:"ritco",
+          password:await bcrypt.hash("phantom123", 12),
+          roleId:1,
+          roleName:"driver",
+          isAssigned:"false",
+          passwordResetToken:"",
+          createdAt:new Date(),
+          updatedAt:new Date(),
+      },
+      {
+        // id:103,
+        uuid:"9e3d2a6c-5484-4d86-8eb0-d0098cd7a540",
+        name:"Armstrong ngororano",
+        gender:"male",
+        idNumber:20040820,
+        district:"kicukiro",
+        sector:"kicukiro",
+        cell:"kicukiro",
+        email:"avengersphantom7d@gmail.com",
+        permitId:"qwerty12",
+        telNumber:"785278453",
+        carplate:"RAB322",
+        capacity:"10",
+        vehicletype:"ritco",
+        password:await bcrypt.hash("phantom123", 12),
+        roleId:1,
+        roleName:"driver",
+        isAssigned:"true",
+        passwordResetToken:"",
+        createdAt:new Date(),
+        updatedAt:new Date(),
+      },
```

Comment:
Calling this seeder file admin.js is misleading, it is a user seeder, not an admin seeder

Reply:
am not the one who created it .all I have done is to edit it

Reply:
am not the one who created it .all I have done is to edit it